jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -234,15 +235,15 @@ def test_parse_asdf_in_fits_4d(self, imviz_helper, tmp_path):
 
     @pytest.mark.remote_data
     def test_parse_jwst_nircam_level2(self, imviz_helper):
-        # TODO: Change back to smaller number (30?) when ITSD is convinced it is them and not us.
-        #       Help desk ticket INC0183598, J. Quick.
-        filename = download_file(self.jwst_asdf_url_1, cache=True, timeout=100)
```

Comment:
This makes me wonder... Do we also need to expose the `timeout` keyword to user API? I think it is an accepted keyword in both `astropy` and `astroquery`.

Reply:
I think it would be good to expose it.